c++/src/kj/test.c++

```diff
@@ -346,6 +346,8 @@ public:
     if (passCount > 0) write(GREEN, kj::str(passCount, " test(s) passed"), "");
     if (failCount > 0) write(RED, kj::str(failCount, " test(s) failed"), "");
     context.exit();
+
+    return true;
```

Comment:
Warning is generated by gcc 5.3.1. Sorry, I did not really read the code. I will try KJ_UNREACHABLE on Monday.

Clang was fine. 

There is one more buggy warning in capnp where gcc warns that kj::Maybe instance may not be initialized. Do you care about warning free gcc build in master?